taipy/gui_core/_adapters.py

```diff
@@ -264,7 +257,7 @@ def _get_datanode_property(attr: str):
 
 
 class _GuiCoreScenarioProperties(_TaipyBase):
-    __SC_TYPES = {
+    _SC_TYPES = {
```

Comment:
I guess you don't need the "_SC" prefix: you're in the `_GuiCoreScenarioProperties` class already.
`_TYPES` is clear enough in my opinion.

Reply:
we also have DN_ for Datanode